### PR TITLE
Add subtopics in the Concepts page

### PR DESCRIPTION
**Pull request by RukshiW**

### CODE REVIEW

File: en/docs/develop/create-api.md [hunk 1]
```diff
@@ -1,2 +1,4 @@
 # Create APIs
```
Comment (shaniR) [inline en/docs/develop/create-api.md:1]: APIs ? shall we rename the file too?

File: en/docs/develop/create-integration.md [hunk 2]
```diff
@@ -1 +1,3 @@
 # Create Integration
```
Comment (shaniR) [inline en/docs/develop/create-integration.md:1]: same here

**Review by shaniR — APPROVED** [review]

File: en/docs/develop/create-api.md (hunk 1 above)
Comment (RukshiW) [reply en/docs/develop/create-api.md]: Done

File: en/docs/develop/create-integration.md (hunk 2 above)
Comment (RukshiW) [reply en/docs/develop/create-integration.md]: Done

**Pull request merged**